README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Did some work with STEP 6 preprocessing data, but I want to revisit to really get my head around it. I understand things like setting a range, setting the mean to 0 and std to 1, I need to understand a little more.